sandbox/index.html

```diff
@@ -12,6 +12,7 @@
       })
       DD_LOGS.init({
         clientToken: 'xxx',
+        enableExperimentalFeatures: ['custom-timings'],
```

Comment:
What is the intent?

Reply:
This is a left over from when I was testing locally, I will remove it.